javascript/react-js/react-introduction.md

```diff
@@ -62,7 +65,7 @@ class App extends Component {
 export default App
 ~~~
 
-Does the code look foreign? Don't worry, the structure isn't as difficult as it may look. Let's walk through it step by step.
+Does the code look foreign? Don't worry, it isn't as difficult as it looks. Let's walk through it step by step.
```

Comment:
```suggestion
Does the code look foreign? Don't worry, it isn't as complicated as it looks. Let's walk through it step by step.
```

I think complicated (or complex) would probably be more suitable? I understand the original intent, but I wonder if my suggestion would be the more common interpretation of the code.